joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

left_wheel_effort_controller:
  type: effort_controllers/JointEffortController
  joint: base_footprint_to_wheel_BL

right_wheel_effort_controller:
  type: effort_controllers/JointEffortController
  joint: base_footprint_to_wheel_BR

left_steer_position_controller:
  type: effort_controllers/JointPositionController
  joint: base_footprint_to_steering_link_left
  pid: {p: 5.0, i: 0.01, d: 0.5}

right_steer_position_controller:
  type: effort_controllers/JointPositionController
  joint: base_footprint_to_steering_link_right
  pid: {p: 5.0, i: 0.01, d: 0.5}
